ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -128,6 +128,18 @@ interface {{ ethernet_interface }}
 {%         if ethernet_interfaces[ethernet_interface].qos.cos is arista.avd.defined %}
    qos cos {{ ethernet_interfaces[ethernet_interface].qos.cos }}
 {%         endif %}
+{%     if ethernet_interfaces[ethernet_interface].priority_flow_control is arista.avd.defined and ethernet_interfaces[ethernet_interface].priority_flow_control.enabled is arista.avd.defined(true) %}
```

Comment:
This can be simplified to `{% if ethernet_interfaces[ethernet_interface].priority_flow_control.enabled is arista.avd.defined(true) %}`. Same below